src/datastructures.apib

```diff
@@ -185,3 +185,10 @@
 ## User (object)
 + type: `user` (string)
 + id: `bef8c53b-b6bc-0fde-944e-5c4b16e5155a` (string)
+
+## CommercialDiscount (object)
++ type: `percentage` (enum)
+    + Members
+        + percentage
++ value: 15.5 (number)
```

Comment:
Q: Useful to keep the label mentioning "Values between 0 and 100"?

Reply:
Indeed, I added it again. Although it does not seem to appear on the documentation.. But our next API documentation framework can maybe handle it

Reply:
Correct, I added it again